<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATL-Electrical Chemistry Analysor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Pos_Box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>310</height>
     </rect>
    </property>
    <property name="title">
     <string>Positive</string>
    </property>
    <widget class="QTabWidget" name="Pos_Tabs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>431</width>
       <height>251</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pos_data">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <widget class="QTableWidget" name="pos_data_table">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>430</width>
         <height>230</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>12</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>16</number>
       </attribute>
       <column>
        <property name="text">
         <string>Voltage</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Capacity</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="pos_VQ">
      <attribute name="title">
       <string>V-Q</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="pos_VQ_layout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="pos_dVdQ">
      <attribute name="title">
       <string>dV/dQ-Q</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-3</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="pos_dVdQ_layout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="pos_QV">
      <attribute name="title">
       <string>Q-V</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="pos_QV_layout">
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="pos_dQdV">
      <attribute name="title">
       <string>dQdV-V</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="pos_dQdV_layout">
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="pos_smooth_method">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>279</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Simple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Savitsky Golay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bezier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kalman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spline</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pos_data_smooth">
     <property name="geometry">
      <rect>
       <x>311</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pos_smooth_param">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>281</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>282</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>284</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>263</y>
       <width>428</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pos_data_reset">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>280</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>277</y>
       <width>3</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>277</y>
       <width>3</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Neg_Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>451</width>
      <height>310</height>
     </rect>
    </property>
    <property name="title">
     <string>Negative</string>
    </property>
    <widget class="QTabWidget" name="Neg_Tabs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>431</width>
       <height>251</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="neg_data">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <widget class="QTableWidget" name="neg_data_table">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>430</width>
         <height>230</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>12</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>16</number>
       </attribute>
       <column>
        <property name="text">
         <string>Voltage</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Capacity</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="neg_VQ">
      <attribute name="title">
       <string>V-Q</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_9">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="neg_VQ_layout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="neg_dVdQ">
      <attribute name="title">
       <string>dV/dQ-Q</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_10">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-3</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="neg_dVdQ_layout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="neg_QV">
      <attribute name="title">
       <string>Q-V</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_11">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="neg_QV_layout">
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="neg_dQdV">
      <attribute name="title">
       <string>dQdV-V</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_12">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="neg_dQdV_layout">
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="neg_smooth_method">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>279</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Simple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Savitsky Golay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bezier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kalman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spline</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="neg_data_smooth">
     <property name="geometry">
      <rect>
       <x>311</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="neg_smooth_param">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>281</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>282</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>284</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>263</y>
       <width>428</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="neg_data_reset">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>280</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>277</y>
       <width>3</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>277</y>
       <width>3</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Full_Box">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>531</width>
      <height>310</height>
     </rect>
    </property>
    <property name="title">
     <string>Full Cell</string>
    </property>
    <widget class="QTabWidget" name="Full_Tabs">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>14</y>
       <width>431</width>
       <height>251</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="full_data">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <widget class="QTableWidget" name="full_data_table">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>430</width>
         <height>230</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>12</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>16</number>
       </attribute>
       <column>
        <property name="text">
         <string>Voltage</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Capacity</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="full_VQ">
      <attribute name="title">
       <string>V-Q</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_13">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="full_VQ_layout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="full_dVdQ">
      <attribute name="title">
       <string>dV/dQ-Q</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_14">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-3</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="full_dVdQ_layout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="full_QV">
      <attribute name="title">
       <string>Q-V</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_15">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="full_QV_layout">
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="full_dQdV">
      <attribute name="title">
       <string>dQdV-V</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_16">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="full_dQdV_layout">
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="full_smooth_method">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>279</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Simple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Savitsky Golay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bezier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kalman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spline</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="full_data_smooth">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="full_smooth_param">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>281</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>282</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>284</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>263</y>
       <width>428</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="full_data_reset">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>257</x>
       <y>277</y>
       <width>3</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>277</y>
       <width>3</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>71</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionPositive_Reference"/>
     <addaction name="actionNegative_Reference"/>
     <addaction name="actionMeasured_Data"/>
    </widget>
    <addaction name="menuLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionPreference"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPositive_Reference">
   <property name="text">
    <string>Positive Reference</string>
   </property>
  </action>
  <action name="actionNegative_Reference">
   <property name="text">
    <string>Negative Reference</string>
   </property>
  </action>
  <action name="actionMeasured_Data">
   <property name="text">
    <string>Measured Data</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
